### PR TITLE
Development/user profile

### PR DESCRIPTION
**Pull request by judeamos**
Changes made: 
- I've removed user data context from the user-profile folder, now we're using the global context in the packages folder. 
- Reflected changes where needed for using global `UserDataContext.js`
- cleaned up code from merging. 
- Changes from master branch have been merged into this one as well.

### CODE REVIEW

File: apps/app/next.config.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ const withTM = require('next-transpile-modules')([
   '@devlaunchers/site-projects',
   '@devlaunchers/dev-recruiters',
   '@devlaunchers/website',
-  '@devlaunchers/user-profile',
```
Comment (Enjoy2Live) [inline apps/app/next.config.js:10]: why remove this? I think it's needed

Comment (Enjoy2Live) [reply apps/app/next.config.js:10]: maybe it was part of a conflict but definitely needed to be back

**Review by Enjoy2Live — CHANGES_REQUESTED** [review]

File: apps/app/next.config.js (hunk 1 above)
Comment (judeamos) [reply apps/app/next.config.js:10]: ` '@devlaunchers/user-profile'` is duplicated, so i deleted one of them. If you expand it you'll see it on top

**Review by Enjoy2Live — APPROVED** [review]
LGTM

**Pull request merged**